### ISSUE TITLE
[console] add commands with keybindings

### ISSUE THREAD

**Issue by akosyakov** [theia-ide/theia#229]
Add commands like `clean` (Ctrl + K), `copy all` (Ctrl + A) and so on, look at other tools.

**Comment by hexa00** [comment]
Let's avoid rebinding readline's keybinding in a console however I use those all the time.

C-k is kill line
C-a is move the the start of the line


See a complete list here (output of `bind -p | grep -F "\C"` in bash)
```
bind -p | grep -F "\C"
"\C-g": abort
"\C-x\C-g": abort
"\e\C-g": abort
"\C-j": accept-line
"\C-m": accept-line
"\C-b": backward-char
"\C-h": backward-delete-char
"\C-?": backward-delete-char
"\C-x\C-?": backward-kill-line
"\e\C-h": backward-kill-word
"\e\C-?": backward-kill-word
"\C-a": beginning-of-line
"\C-xe": call-last-kbd-macro
"\C-]": character-search
"\e\C-]": character-search-backward
"\C-l": clear-screen
"\C-i": complete
"\C-d": delete-char
"\C-x\C-v": display-shell-version
"\C-xA": do-lowercase-version
"\C-xB": do-lowercase-version
"\C-xC": do-lowercase-version
"\C-xD": do-lowercase-version
"\C-xE": do-lowercase-version
"\C-xF": do-lowercase-version
"\C-xG": do-lowercase-version
"\C-xH": do-lowercase-version
"\C-xI": do-lowercase-version
"\C-xJ": do-lowercase-version
"\C-xK": do-lowercase-version
"\C-xL": do-lowercase-version
"\C-xM": do-lowercase-version
"\C-xN": do-lowercase-version
"\C-xO": do-lowercase-version
"\C-xP": do-lowercase-version
"\C-xQ": do-lowercase-version
"\C-xR": do-lowercase-version
"\C-xS": do-lowercase-version
"\C-xT": do-lowercase-version
"\C-xU": do-lowercase-version
"\C-xV": do-lowercase-version
"\C-xW": do-lowercase-version
"\C-xX": do-lowercase-version
"\C-xY": do-lowercase-version
"\C-xZ": do-lowercase-version
"\e\C-i": dynamic-complete-history
"\C-x\C-e": edit-and-execute-command
"\C-x)": end-kbd-macro
"\C-e": end-of-line
"\C-x\C-x": exchange-point-and-mark
"\C-f": forward-char
"\C-s": forward-search-history
"\C-x*": glob-expand-word
"\C-xg": glob-list-expansions
"\C-k": kill-line
"\C-n": next-history
"\C-o": operate-and-get-next
"\C-x!": possible-command-completions
"\C-x/": possible-filename-completions
"\C-x@": possible-hostname-completions
"\C-x~": possible-username-completions
"\C-x$": possible-variable-completions
"\C-p": previous-history
"\C-q": quoted-insert
"\C-v": quoted-insert
"\C-x\C-r": re-read-init-file
"\C-r": reverse-search-history
"\e\C-r": revert-line
"\C-@": set-mark
"\e\C-e": shell-expand-line
"\C-x(": start-kbd-macro
"\C-t": transpose-chars
"\C-x\C-u": undo
"\C-_": undo
"\C-u": unix-line-discard
"\C-w": unix-word-rubout
"\C-y": yank
"\e\C-y": yank-nth-arg
```

**Comment by kittaakos** [comment]
> C-k is kill line
> C-a is move the the start of the line

On OS X these are with the Control instead of Cmd, and I also use them all the time. Probably we need to check what is done by VSCode and we might need platform specific bindings. Also, we have to make sure if any of the bindings conflict with a browser default.

**Comment by hexa00** [comment]
Maybe @akosyakov wanted to say Cmd instead of Ctrl in his description then ? If so that's fine with me...

But yes I guess we would need to create a map of those default bindings in the wiki or something or reference to them

**Comment by hexa00** [comment]
I think is is fixed with: #814 

Please re-open if I missed something
